src/tcgwars/logic/impl/gen4/Arceus.groovy

```diff
@@ -128,12 +128,12 @@ public enum Arceus implements LogicCardInfo {
   ULTIMATE_ZONE_91 ("Ultimate Zone", "91", Rarity.UNCOMMON, [TRAINER]),
   DOME_FOSSIL_92 ("Dome Fossil", "92", Rarity.COMMON, [TRAINER]),
   HELIX_FOSSIL_93 ("Helix Fossil", "93", Rarity.COMMON, [TRAINER]),
-  ARCEUS_LV_X_94 ("Arceus LV.X", "94", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _COLORLESS_]),
-  ARCEUS_LV_X_95 ("Arceus LV.X", "95", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _COLORLESS_]),
-  ARCEUS_LV_X_96 ("Arceus LV.X", "96", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _COLORLESS_]),
-  GENGAR_LV_X_97 ("Gengar LV.X", "97", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _PSYCHIC_]),
-  SALAMENCE_LV_X_98 ("Salamence LV.X", "98", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _COLORLESS_]),
-  TANGROWTH_LV_X_99 ("Tangrowth LV.X", "99", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _GRASS_]),
+  ARCEUS_LV_X_94 ("Arceus", "94", Rarity.HOLORARE, [LEVEL_UP, POKEMON, _COLORLESS_]),
+  ARCEUS_LV_X_95 ("Arceus", "95", Rarity.HOLORARE, [LEVEL_UP, POKEMON, _COLORLESS_]),
+  ARCEUS_LV_X_96 ("Arceus", "96", Rarity.HOLORARE, [LEVEL_UP, POKEMON, _COLORLESS_]),
+  GENGAR_LV_X_97 ("Gengar", "97", Rarity.HOLORARE, [LEVEL_UP, POKEMON, _PSYCHIC_]),
+  SALAMENCE_LV_X_98 ("Salamence", "98", Rarity.HOLORARE, [LEVEL_UP, POKEMON, _COLORLESS_]),
+  TANGROWTH_LV_X_99 ("Tangrowth", "99", Rarity.HOLORARE, [LEVEL_UP, POKEMON, _GRASS_]),
```

Comment:
I think cards should keep their names with LV.X. The DeckService validity check should be updated.

Reply:
How would we handle that Break Pokemon can evolve from LvX's?

Reply:
I think the easiest solution is just to remove Lv.X from the name.  It also works out because you can only have a combination of 4 of LvX + Non-LvX Pokemon

Reply:
> How would we handle that Break Pokemon can evolve from LvX's?

Is this a thing? Wow. I did not know. 

I think it's best to keep them here and change `PokemonCardSet.getName()` method to drop LV.X suffix before returning. That will make it work with evolution checks. Otherwise the deck builder/import deck features/in-game logging will plain card name which can be confusing.

Reply:
I've modified `PokemonCardSet.getName()` accordingly